nltk/test/unit/translate/test_bleu.py

```diff
@@ -174,8 +180,9 @@ def test_reference_or_hypothesis_shorter_than_fourgrams(self):
         # is shorter than 4.
         references = ['let it go'.split()]
         hypothesis = 'let go it'.split()
-        # Checks that the value the hypothesis and reference returns is 1.0
-        assert(sentence_bleu(references, hypothesis) == 1.0)
+        # Checks that the value the hypothesis and reference returns is 0.0
+        # exp(w_1 * 1 * w_2 * 1 * w_3 * 1 * w_4 * -inf) = 0
+        self.assertAlmostEqual(sentence_bleu(references, hypothesis), 0.0, places=4)
```

Comment:
This is the ONLY test case that your hot fix should be solving. 

I agree that the original 1.0 is not perfect but changing it to 0.0 isn't acceptable too =(